### PR TITLE
Oct news additions

### PR DESCRIPTION
**Pull request by hauten**
Updated a few older event posts
Fixed a typo
Added new releases
@tzanio - please review 2020-09-30-ceed-ms35.md for accuracy

### CODE REVIEW

File: _posts/2020-09-30-ceed-ms35.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "CEED M-35 Milestone Report Released"
```
Comment (tzanio) [inline _posts/2020-09-30-ceed-ms35.md:2]:
```suggestion
title: "CEED MS-35 Milestone Report Released"
```

File: _posts/2020-09-30-ceed-ms35.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+categories: release
+---
+
+The Center for Efficient Exascale Discretizations ([CEED](https://github.com/CEED)) is part of the Department of Energy’s [Exascale Computing Project (ECP(https://www.exascaleproject.org/))]. The Center helps applications leverage future architectures by developing discretization algorithms that better exploit the hardware and deliver a significant performance gain over conventional methods. CEED regularly releases milestone (MS) [reports](https://ceed.exascaleproject.org/pubs/) to demonstrate progress and ensure transparency throughout its multi-year efforts.
```
Comment (tzanio) [inline _posts/2020-09-30-ceed-ms35.md:6]:
```suggestion
The Center for Efficient Exascale Discretizations ([CEED](https://github.com/CEED)) is part of the Department of Energy’s Exascale Computing Project ([ECP](https://www.exascaleproject.org/)). The Center helps applications leverage future architectures by developing discretization algorithms that better exploit the hardware and deliver a significant performance gain over conventional methods. CEED regularly releases milestone (MS) [reports](https://ceed.exascaleproject.org/pubs/) to document progress and ensure transparency throughout its multi-year efforts.
```

File: _posts/2020-09-30-ceed-ms35.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+The [MS-35 report](https://ceed.exascaleproject.org/docs/ceed-ms35-report.pdf) details the following goals:
```
Comment (tzanio) [inline _posts/2020-09-30-ceed-ms35.md:8]:
```suggestion
The [MS-35 report](https://ceed.exascaleproject.org/docs/ceed-ms35-report.pdf) details the CEED accomplishments towards the following goals:
```

File: _posts/2020-09-30-ceed-ms35.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- Work with vendors to port and run CEED benchmarks on early access systems for Aurora (Argonne National Lab) and Frontier (Oak Ridge National Lab)
+- Make the best BP/BK implementations from Nek, MFEM, libParanumal, and the external community available in [libCEED](https://github.com/CEED/libCEED)
```
Comment (tzanio) [inline _posts/2020-09-30-ceed-ms35.md:10]:
```suggestion
- Make the best CEED benchmark implementations from Nek5000, MFEM, libParanumal, and the external community available in [libCEED](https://github.com/CEED/libCEED)
```

File: _posts/2020-09-30-ceed-ms35.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- Organize the next CEED Annual meeting ([CEED4AM](https://ceed.exascaleproject.org/ceed4am/))
+- Optimize CEED applications and miniapps for Aurora and Frontier architectures
```
Comment (tzanio) [inline _posts/2020-09-30-ceed-ms35.md:12]:
```suggestion
- Optimize the CEED applications and miniapps for the Aurora and Frontier architectures
```

**Review by tzanio — APPROVED** [review]
Thanks so much @hauten !

**Review by IanLee1521 — APPROVED** [review]
LGTM!

Thanks Holly!

File: _posts/2020-09-30-ceed-ms35.md (hunk 5 above)
Comment (hauten) [reply _posts/2020-09-30-ceed-ms35.md:12]: New commit 92e9737

File: _posts/2020-09-30-ceed-ms35.md (hunk 4 above)
Comment (hauten) [reply _posts/2020-09-30-ceed-ms35.md:10]: New commit 92e9737

File: _posts/2020-09-30-ceed-ms35.md (hunk 3 above)
Comment (hauten) [reply _posts/2020-09-30-ceed-ms35.md]: New commit 92e9737

File: _posts/2020-09-30-ceed-ms35.md (hunk 2 above)
Comment (hauten) [reply _posts/2020-09-30-ceed-ms35.md]: New commit 92e9737

File: _posts/2020-09-30-ceed-ms35.md (hunk 5 above)
Comment (hauten) [reply _posts/2020-09-30-ceed-ms35.md]: New commit 1555bfe

**Pull request merged**